package.json

```diff
@@ -2,9 +2,11 @@
   "name": "egghead-instructor-center",
   "version": "1.0.0",
   "scripts": {
-    "dev": "NODE_PATH=./src react-scripts start",
+    "dev": "REACT_APP_EGGHEAD_BASE_URL=http://egghead.dev:5000 NODE_PATH=./src react-scripts start",
+    "dev:staging": "REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com NODE_PATH=./src react-scripts start",
+    "dev:prod": "REACT_APP_EGGHEAD_BASE_URL=https://egghead.io NODE_PATH=./src react-scripts start",
     "test": "NODE_PATH=./src react-scripts test --env=jsdom",
-    "build": "NODE_PATH=./src react-scripts build",
+    "build": "REACT_APP_EGGHEAD_BASE_URL=https://egghead.io NODE_PATH=./src react-scripts build",
     "verify": "CI=true yarn test && yarn build"
   },
```

Comment:
I'm sure there is a way to DRY this up with npm `config` variables or something; I'll probably do that at some point but this works for now.